doc-content/enterprise-only/openshift/operator-deploy-basic-proc.adoc

```diff
@@ -21,7 +21,9 @@ In {PRODUCT} {ENTERPRISE_VERSION}, high-availability {CENTRAL} functionality dep
 ====
 +
 ** `rhpam-production`: An environment for running existing services for staging and production purposes. This environment includes {CENTRAL} Monitoring, Smart Router, and two groups of {KIE_SERVER} pods. You can deploy and undeploy services on every such group and also scale the group up or down as necessary. Use {CENTRAL} Monitoring to deploy, run, and stop the services and to monitor their execution.
-** `rhpam-production-immutable`: An alternate environment for running existing services for staging and production purposes. This environment includes {CENTRAL} Monitoring. You can configure one or more {KIE_SERVER} pods that build a service from source and replicate each pod as necessary. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image to replace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
+** `rhpam-production-immutable`: An alternate environment for running existing services for staging and production purposes. You can configure one or more {KIE_SERVER} pods that build a service from source and replicate each pod as necessary. In this environment, when you deploy a {KIE_SERVER} pod, it builds an image that loads and starts a service or group of services. You cannot stop any service on the pod or add any new service to the pod. If you want to use another version of a service or modify the configuration in any other way, you deploy a new server image to replace the old one. In this system, the {KIE_SERVER} runs like any other pod on the OpenShift environment. You can use any container-based integration workflows and you do not need to use any other tools to manage the pods.
++
+Optionally, you can include {CENTRAL} Monitoring in this environment to monitor, stop, and restart the execution of services. 
```

Comment:
you can include {CENTRAL}  Monitoring  by explicit configure in in the Wizard. in this environment to monitor


execution of services.  -> Business Process.